src/TAction.cpp

```diff
@@ -138,7 +138,7 @@ bool TAction::compileScript()
     mFuncName = QString("Action")+QString::number( mID );
     QString code = QString("function ")+ mFuncName + QString("()\n") + mScript + QString("\nend\n");
     QString error;
-    if( mpHost->mLuaInterpreter.compile( code, error ) )
+    if( mpHost->mLuaInterpreter.compile( code, error, QString("Button: ") + getName() ) )
```

Comment:
You may be right - I may have got the context wrong so it would be:

    QStringLiteral( "Button: %1").arg( getName() );

instead to make a `QString` that is **not** subject to the translation efforts of _QtLinguist_.